homeassistant/components/lg_soundbar/const.py

```diff
@@ -0,0 +1,5 @@
+"""Constants for the LG Soundbar integration."""
+DOMAIN = "lg_soundbar"
+
+DEFAULT_HOST = "192.168.1.32"
```

Comment:
Remove this. This is an IP that is just for your set up.

Reply:
i did think of removeing this but i had a look and a few integrations have a DEFAULT_HOST so i didnt take it out, but now i have removed it